onnxruntime/test/providers/nnapi/nnapi_basic_test.cc

```diff
@@ -4,7 +4,10 @@
 #if !defined(ORT_MINIMAL_BUILD) || defined(ORT_EXTENDED_MINIMAL_BUILD)
 #include "core/common/logging/logging.h"
 #include "core/providers/nnapi/nnapi_builtin/nnapi_execution_provider.h"
+#include "core/providers/nnapi/nnapi_builtin/nnapi_lib/NeuralNetworksTypes.h"
+#include "core/providers/nnapi/nnapi_builtin/nnapi_lib/nnapi_implementation.h"
```

Comment:
are these headers needed?

Reply:
They are used for `const auto* nnapi = NnApiImplementation();
  if (nnapi->nnapi_runtime_feature_level < ANEURALNETWORKS_FEATURE_LEVEL_3)` to get nnapi runtime android feature level.

Reply:
ah ok, I missed that